Model/AcuraNSX.py

```diff
@@ -0,0 +1,49 @@
+from Model.Vehicle import Vehicle
+from Model.Car import Car
+from enum import IntEnum
+from Model.Car import Transmission
+
+class AcuraNSX(Car):
+    def __init__(
+            self, 
+            name: str = "Acura NSX", 
+            color: str = "red", 
+            speed: float = 5, 
+            width: float = 156., 
+            height: float = 200.,
+            num_doors: int = 2, 
+            transmission: Transmission = Transmission.MANUAL,
+            *args,
+            **kwargs,
+        ):
```

Comment:
Be careful with positional arguments, these are in the wrong order. To avoid confusion, it is a good practice to use keyword arguments instead so that if the position of the arguments change your functions calls will not need to be changed. Failing to do so can cause logic errors that are nearly impossible to track down.

Reply:
expected order is as follows:

name: Any,
color: str,
speed: float,
width: float,
height: float,
num_doors: int,
transmission: Transmission,

Reply:
Since width was passed in for speed, your speed was set to 158. Hence why you had to adjust your lists with a constant 30 to get a good speed.